firstvoices/backend/tasks/visibility_tasks.py

```diff
@@ -8,22 +9,30 @@
 from backend.search.tasks.site_content_indexing_tasks import (
     sync_all_site_content_in_indexes,
 )
+from backend.tasks.utils import ASYNC_TASK_END_TEMPLATE, ASYNC_TASK_START_TEMPLATE
 
 
 @shared_task
 def bulk_change_visibility(job_instance_id):
     """
     Changes the visibility of all site content from one visibility to another.
     """
+
+    logger = get_task_logger(__name__)
+    logger.info(ASYNC_TASK_START_TEMPLATE, f"job_instance_id: {job_instance_id}")
+
     job = BulkVisibilityJob.objects.get(id=job_instance_id)
     site = job.site
     indexing_paused_feature = SiteFeature.objects.get_or_create(
         site=site, key="indexing_paused"
     )
 
     if BulkVisibilityJob.objects.filter(status=JobStatus.STARTED, site=site).exists():
+        cancelled_message = "Job cancelled as another bulk visibility job is already in progress for the same site."
         job.status = JobStatus.CANCELLED
-        job.message = "Job cancelled as another bulk visibility job is already in progress for the same site."
+        job.message = cancelled_message
+        logger.info(cancelled_message)
+        logger.info(ASYNC_TASK_END_TEMPLATE)
         job.save()
```

Comment:
The save should happen before the two log messages, I think. Otherwise we risk having inaccurate logs if something goes wrong with the save.